src/ansys/geometry/core/designer/component.py

```diff
@@ -484,6 +487,107 @@ def extrude_sketch(
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    def create_sweeping_profile(
```

Comment:
@RobPasMue to keep in line with the rest of the library ("extrude_sketch") should we name this method `sweep_sketch()` and the other method either `sweep_chain()` or `sweep_curves()`?

Reply:
I agree - let's keep thing in line. Naming convention should be ``sweep_*``